JSCS plugins fail to load after upgrade to 3.4.4

I just upgraded to version 3.4.4 of linter-jscs and it's crashing because it can't find my JSCS plugins. Unfortunately this crash occurs on every save (lint attempt). This is definitely a regression, I'm certain it did work with the previous version I had.

Here is the stack:
```
AssertionError: `plugin` should be a function
    at NodeConfiguration.Configuration._loadPlugin (/Users/wesley/.atom/packages/linter-jscs/node_modules/jscs/lib/config/configuration.js:571:5)
    at NodeConfiguration._loadPlugin (/Users/wesley/.atom/packages/linter-jscs/node_modules/jscs/lib/config/node-configuration.js:138:48)
    at NodeConfiguration.<anonymous> (/Users/wesley/.atom/packages/linter-jscs/node_modules/jscs/lib/config/configuration.js:495:18)
    at Array.forEach (native)
    at NodeConfiguration.Configuration._processConfig (/Users/wesley/.atom/packages/linter-jscs/node_modules/jscs/lib/config/configuration.js:494:25)
    at NodeConfiguration.Configuration.load (/Users/wesley/.atom/packages/linter-jscs/node_modules/jscs/lib/config/configuration.js:238:10)
    at [object Object].StringChecker.configure (/Users/wesley/.atom/packages/linter-jscs/node_modules/jscs/lib/string-checker.js:67:29)
    at [object Object].Checker.configure (/Users/wesley/.atom/packages/linter-jscs/node_modules/jscs/lib/checker.js:28:39)
    at Object.lint (/Users/wesley/.atom/packages/linter-jscs/src/linter-jscs.js:123:19)
    at promises.push.Promise.then._this.emitter.emit.linter (/Users/wesley/.atom/packages/linter/lib/linter-registry.js:63:26)
    at /Users/wesley/.atom/packages/linter/lib/linter-registry.js:62:23
    at Set.forEach (native)
    at LinterRegistry.lint (/Users/wesley/.atom/packages/linter/lib/linter-registry.js:60:18)
    at /Users/wesley/.atom/packages/linter/lib/linter.coffee:98:16
    at Function.module.exports.Emitter.simpleDispatch (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/event-kit/lib/emitter.js:25:14)
    at Emitter.module.exports.Emitter.emit (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/event-kit/lib/emitter.js:125:28)
    at /Users/wesley/.atom/packages/linter/lib/editor-linter.js:80:22
    at /Users/wesley/.atom/packages/linter/lib/helpers.coffee:23:18
```

And a screenshot:
<img width="464" alt="screen shot 2016-02-06 at 1 18 11 pm" src="https://cloud.githubusercontent.com/assets/825487/12868410/266f3ee4-ccd4-11e5-936a-ba8eaa376b55.png">

I started to debug this and discovered a few things. It seems related to the plugin that's defined in my `.jscsrc` file. I believe the issue is with the linter-jscs's `options`. See this screenshot:

![image](https://cloud.githubusercontent.com/assets/825487/12868607/ba44615a-ccd7-11e5-89da-96866fa88c75.png)

This is just one part of the error stack, but you can see the problem clearly is the `configPath` is missing. If you go to the next layer in the stack you'll find that without a `configPath` it's impossible to find and load a plugin ([node-configuration.js](https://github.com/jscs-dev/node-jscs/blob/5130e5e852ab3dd4dd70b30391e5477b709caf5c/lib/config/node-configuration.js#L131-L137)). 

-------------

Is there any easy way for me to downgrade to the previous version of linter-jscs so I can restore the functionality for now?

--- a/src/linter-jscs.js
+++ b/src/linter-jscs.js
@@ -120,7 +120,13 @@ export default class LinterJSCS {
         // Options passed to `jscs` from package configuration
         const options = { esnext: this.esnext, preset: this.preset };
 
-        this.jscs.configure(overrideOptions || Object.assign({}, options, config));
+        // `configPath` is non-enumerable so `Object.assign` won't copy it.
+        // Without a proper `configPath` JSCS plugs cannot be loaded. See #175.
+        let jscsConfig = overrideOptions || Object.assign({}, options, config);
+        if (!jscsConfig.configPath && config) {
+          jscsConfig.configPath = config.configPath;
+        }
+        this.jscs.configure(jscsConfig);
 
         // We don't have a config file present in project directory
         // let's return an empty array of errors

